### PR TITLE
New brb 'I need a break from the screen'

### PR DESCRIPTION
**Pull request by Daiozen**
Self-explanatory

### CODE REVIEW

File: Monika After Story/game/script-brbs.rpy [hunk 1]
```diff
@@ -671,6 +671,83 @@ label monika_idle_working_callback:
...
+
+label monika_idle_screen_break_callback:
+    $ wb_quip = mas_brbs.get_wb_quip()
+    if mas_isMoniNormal(higher=True):
+        
+        m 1eub "Welcome back, [player]."
+        
+        if mas_brbs.was_idle_for_at_least(datetime.timedelta(minutes=30), "monika_idle_screen_break"):
+            m 1hksdlb "You must've really needed that break, considering how long you were gone."
+            m 1eka "I hope you're feeling a little more rested now."
```
Comment (multimokia) [inline Monika After Story/game/script-brbs.rpy:734]: Unsure if `rested` is the right word here
```suggestion
            m 1eka "I hope you're feeling a little more rested now."
```

File: Monika After Story/game/script-brbs.rpy [hunk 2]
```diff
@@ -671,6 +671,83 @@ label monika_idle_working_callback:
...
+
+label monika_idle_screen_break:
+    if mas_isMoniNormal(higher=True):
+
+        if mas_getEV('monika_idle_screen_break').timePassedSinceLastSeen_dt(mas_getSessionLength()):
+
+            if mas_getSessionLength() < datetime.timedelta(minutes=40):
+                m 1esc "Oh,{w=0.3} okay."
+                m 1dsu "You've not been here for that long but if you say you need a break, then you need a break."
+            elif mas_getSessionLength() < datetime.timedelta(minutes=150):
```
Comment (multimokia) [inline Monika After Story/game/script-brbs.rpy:695]: Some clarity
```suggestion
            elif mas_getSessionLength() < datetime.timedelta(hours=2, minutes=30):
```

File: Monika After Story/game/script-brbs.rpy [hunk 3]
```diff
@@ -671,6 +671,83 @@ label monika_idle_working_callback:
...
+    $ wb_quip = mas_brbs.get_wb_quip()
+    if mas_isMoniNormal(higher=True):
+        
+        m 1eub "Welcome back, [player]."
+        
+        if mas_brbs.was_idle_for_at_least(datetime.timedelta(minutes=30), "monika_idle_screen_break"):
+            m 1hksdlb "You must've really needed that break, considering how long you were gone."
+            m 1eka "I hope you're feeling a little more rested now."
+        else:
+            m 1hua "I hope you're feeling a little more rested now~"
```
Comment (multimokia) [inline Monika After Story/game/script-brbs.rpy:736]: Same regarding rested

File: Monika After Story/game/script-brbs.rpy [hunk 4]
```diff
@@ -671,6 +671,83 @@ label monika_idle_working_callback:
...
+
+    else:
+        m 6ckc "..."
+
+    $ mas_idle_mailbox.send_idle_cb("monika_idle_screen_break_callback")
+    $ persistent._mas_idle_data["monika_idle_screen_break"] = True
+    return "idle"
+
+label monika_idle_screen_break_callback:
+    $ wb_quip = mas_brbs.get_wb_quip()
```
Comment (multimokia) [inline Monika After Story/game/script-brbs.rpy:727]: You only use `wb_quip` in the normal+ flow, you can just define this in that flow so we're not getting a quip needlessly potentially.

File: Monika After Story/game/script-brbs.rpy (hunk 1 above)
Comment (Daiozen) [reply Monika After Story/game/script-brbs.rpy:734]: `m 1eka "I hope you're feeling a little better now."` ?

Comment (multimokia) [reply Monika After Story/game/script-brbs.rpy:734]: Sounds good to me

**Pull request merged**